<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat_interface</class>
 <widget class="QDialog" name="chat_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="chat_list">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>31</y>
     <width>671</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="prompt_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>661</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>660</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="loadingBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>710</y>
     <width>651</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
